stdlib/2and3/zipimport.pyi

```diff
@@ -1,18 +1,25 @@
 """Stub file for the 'zipimport' module."""
 
-from typing import Optional
+import sys
+from typing import List, Optional, Tuple
 from types import CodeType, ModuleType
+if sys.version_info >= (3, 7):
+    from importlib.abc import ResourceReader
 
 class ZipImportError(ImportError): ...
 
 class zipimporter(object):
     archive: str
     prefix: str
-    def __init__(self, archivepath: str) -> None: ...
-    def find_module(self, fullname: str, path: str = ...) -> Optional[zipimporter]: ...
+    def __init__(self, path: str) -> None: ...
+    if sys.version_info >= (3,):
+        def find_loader(self, fullname: str, path: Optional[str] = ...) -> Tuple[Optional[zipimporter], List[str]]: ...  # undocumented
```

Comment:
The docstring suggests the return type is `Union[zipimporter, str, None]`, but it's incorrect.

Reply:
Yeah, I think it would be clearer as `(self, str or None)` instead of `self, str or None`